--- Framework/Computer/WshShell.cs.orig	Thu Nov 15 13:31:54 2012
+++ Framework/Computer/WshShell.cs	Thu Nov 15 13:33:16 2012
@@ -151,8 +151,7 @@
                 {
                     if (!System.IO.File.Exists(this.IconLocation))
                     {
-                        Log.LogError(string.Format(CultureInfo.InvariantCulture, "IconLocation: {0} does not exist.", this.IconLocation));
-                        return;
+                        Log.LogWarning(string.Format(CultureInfo.InvariantCulture, "IconLocation: {0} does not exist.", this.IconLocation));
                     }
 
                     shortcutToCreate.IconLocation = this.IconLocation;
@@ -162,8 +161,7 @@
                 {
                     if (!System.IO.Directory.Exists(this.WorkingDirectory))
                     {
-                        Log.LogError(string.Format(CultureInfo.InvariantCulture, "WorkingDirectory: {0} does not exist.", this.WorkingDirectory));
-                        return;
+                        Log.LogWarning(string.Format(CultureInfo.InvariantCulture, "WorkingDirectory: {0} does not exist.", this.WorkingDirectory));
                     }
 
                     shortcutToCreate.WorkingDirectory = this.WorkingDirectory;
